articles/relnotes/index.md

```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.x
+
+*Release date: October 30th, 2019*
+
+This release contains the following:
+
+- IQ# and Python host functionality can now be installed using conda
+- Updates to Visual Studio Code & Visual Studio extensions to deploy language server as a self-contained executable, no longer requiring the .NET Core SDK to be installed  
+- Migration to .NET Core 3.0
+- Breaking change to Microsoft.Quantum.Simulation.Core.IOperationFactory with introduction of new `Fail` method. It affects only custom simulators that do not extend SimulatorBase. For more details, [view the pull request on GitHub](https://github.com/microsoft/qsharp-runtime/pull/59).
+
```

Comment:
Do you want to add that we actually support Deprecated attributes? Not that there are a lot of people maintaining Q# packages where it would make sense to properly deprecate stuff - but maybe we should not assume that in our release notes.

Reply:
sounds good, I'll add it.